### PR TITLE
[MRG] Adding team portrait

### PR DESCRIPTION
**Pull request by michelfernandez**
Dominik, Joao, Rainer

### CODE REVIEW

**Comment by betatim** [comment]
https://deploy-preview-161--skribble.netlify.com/team/ to see the new team members. Going to be great.

File: site/content/english/team.md [hunk 1]
```diff
@@ -88,13 +88,34 @@ groups:
           retina_portrait: team/patrick-meade-p@2x
           regular_landscape: team/patrick-meade-l
           retina_landscape: team/patrick-meade-l@2x
+      - name: Dominik Kumpf
+        position: Engineering & Testing
+        images:
+          regular_portrait: team/dominik-kumpf-p
+          retina_portrait: team/dominik-kumpf-p@2x
+          regular_landscape: team/dominik-kumpf-p
```
Comment (fursinger) [inline site/content/english/team.md]: Bei `regular_landscape` und `retina_landscape` sollten die Landscape Bilder geladen werden.

File: site/content/english/team.md [hunk 2]
```diff
@@ -88,13 +88,34 @@ groups:
...
           regular_portrait: team/luc-lippuner-l
           retina_portrait: team/luc-lippuner-l@2x
           regular_landscape: team/luc-lippuner-l
           retina_landscape: team/luc-lippuner-l@2x
+      - name: João Bárcia
+        position: Engineering
+        images:
+          regular_portrait: team/joao-barcia-l
+          retina_portrait: team/joao-barcia-l@2x
+          regular_landscape: team/joao-barcia-p
```
Comment (fursinger) [inline site/content/english/team.md]: Siehe Kommentar bei Dominik. Auch hier die Landscape Bilder referenzieren.

**Comment by michelfernandez** [comment]
Actually some people have already looked at it and found nothing more. Actually yes, but I sent it to Till to go through the alt tags and metadata. Think we can merge it over the day.

**Comment by fursinger** [comment]
@dkumpf1 ready for testing.

@RainerGoermann gets the universal portrait for now.

**Comment by dkumpf1** [comment]
Looks good 👍 

Tested: OK

**Pull request merged**